pages/1.12/cli/command-reference/dcos-config/dcos-config-set/index.md

```diff
@@ -9,37 +9,42 @@ enterprise: false
 
 # Description
 
-The `dcos config set` command will add or set DC/OS configuration properties. Table 1 show the available properties.
-
-# Options
-
-| Name  | Description |
-|-----------------------|------------------------------------------------|
-| `core.dcos_acs_token`   | The DC/OS authentication token. When you log into the DC/OS CLI using `dcos auth login`, it stores the authentication token value locally. For more information, see [the IAM API](/1.12/security/oss/iam-api/). |
-| `core.dcos_url`         | The public master URL of your DC/OS cluster|
-| `core.mesos_master_url` | The Mesos master URL. Defaults to `core.dcos_url` |
-| `core.pagination`       | Indicates whether to paginate output. Defaults to true.|
-| `core.ssl_verify`       | Indicates whether to verify SSL certificates or set the path to the SSL certificates|
-| `core.timeout`          | The request timeout in seconds, with a minimum value of 1 second. Defaults to 3 minutes.|
+The `dcos config set` command will add or set a property in the configuration file used for the current cluster. Table 1 shows the available properties.
 
 # Usage
 
 ```bash
-dcos config set <name> <value> [OPTION]
+dcos config set <name> <value> [flags]
 ```
+# Options
+
+| Name |  Description |
+|---------|-------------|
+| `--help, h`   |   Display usage. |
 
-# Positional arguments
+## Positional arguments
 
 | Name |  Description |
 |---------|-------------|
 | `<name>`   |  The name of the property |
 | `<value>`   |   The value of the property |
 
-# Parent command
+### Table 1 - Properties
+
+
+| Name  | Value |
+|-----------------------|------------------------------------------------|
+| `core.dcos_acs_token`   | The DC/OS authentication token. When you log into the DC/OS CLI using `dcos auth login`, it stores the authentication token value locally. For more information, see [the IAM API](/1.12/security/oss/iam-api/). |
+| `core.dcos_url`         | The public master URL of your DC/OS cluster|
+| `core.mesos_master_url` | The Mesos master URL. Defaults to `core.dcos_url` |
+| `core.pagination`       | Indicates whether to paginate output. Defaults to true.|
+| `core.ssl_verify`       | Indicates whether to verify SSL certificates or set the path to the SSL certificates|
+| `core.timeout`          | The request timeout in seconds, with a minimum value of 1 second. Defaults to 3 minutes.|
+| `core.ssh_user` |   |
+| `core.ssh_proxy_ip`  |   |
+|  `core.reporting` | |
```

Comment:
I sadly have not figured out the last two here, maybe @ArmandGrillet can help us out or knows somebody?

Reply:
`ssh_proxy_ip`: Whether to use a fixed ssh proxy host (Bastion) for node SSH access.
`reporting`: Whether to report usage events to Mesosphere.